### ISSUE TITLE
Update scraper to pull date filed

### ISSUE THREAD

**Issue by emilyrbartha** [open-austin/azure-indigent-defense#46]
`charge_date` in the dataset refers to the date of the offense the defendant is being charged for. Can we change the name of `charge_date` to `charge_offense_date` and add the date filed as `charge_filed_date`? The charge filed date will need to be added to the scraper.

**Comment by lianilychee** [comment]
Validate with Josh if still needed. Might be blocked by #57 , but unsure.  Reassign as necessary.

**Comment by normaljosh** [comment]
@nicolassaw could add this once he understands the parsing.. couldn't hurt but low priority
